src/frequenz/sdk/actor/power_distributing/_battery_status.py

```diff
@@ -352,6 +354,26 @@ def _get_current_status(self) -> Status:
 
         return Status.WORKING
 
+    def _is_capacity_present(self, msg: BatteryData) -> bool:
+        """Check whether the battery capacity is NaN or not.
+
+        If battery capacity is missing, then we can't work with it.
+
+        Args:
+            msg: battery message
+
+        Returns:
+            True if battery capacity is missing, false otherwise.
```

Comment:
Shouldn't it be the other way around? or `True if battery capacity is present(not missing)`?

Reply:
@ela-kotulska-frequenz is on sick leave until further notice so I'm not sure who can give you feedback on that.

Reply:
Done